modules/develop/pages/consume-data/consumer-offsets.adoc

```diff
@@ -4,20 +4,125 @@
 // tag::single-source[]
 :description: pass:q[Redpanda uses an internal topic, `__consumer_offsets`, to store committed offsets from each Kafka consumer that is attached to Redpanda.]
 
-Redpanda supports `pass:[__consumer_offsets]`, which is a private topic on a Redpanda broker. The `pass:[__consumer_offsets]` topic stores committed offsets from each Kafka consumer that is attached to Redpanda.
+== Overview
 
-Redpanda exposes the `pass:[__consumer_offsets]` key to enable the many tools in the Kafka ecosystem that rely on this value for their operation, providing greater ecosystem interoperability with environments and applications.
+All messages within Redpanda are organized by glossterm:topic[]. Messages within a topic are divided between partitions, where each glossterm:partition[] holds a subset of the messages. For example, if a topic has 5 partitions, it would be likely (though not guaranteed) that each partition holds about 20% of the messages.
```

Comment:
```suggestion
In Redpanda, all messages are organized by glossterm:topic[] and distributed across multiple partitions. Each glossterm:partition[] holds a subset of the messages. For example, if a topic has five partitions, it would be likely (though not guaranteed) that each partition holds about 20% of the messages.
```